### PR TITLE
adding base,styles,shared sections in resolution block

### PR DESCRIPTION
**Pull request by bhatiaPankaj**

### CODE REVIEW

File: swc-uxp-starter/package.json [hunk 1]
```diff
@@ -29,6 +29,9 @@
   "resolutions": {
     "@spectrum-web-components/theme": "0.14.9",
     "@spectrum-web-components/banner": "0.9.2",
-    "@spectrum-web-components/link": "0.14.1"
+    "@spectrum-web-components/link": "0.14.1",
+    "@spectrum-web-components/shared": "0.15.4",
+    "@spectrum-web-components/styles": "0.22.1",
+    "@spectrum-web-components/base": "0.7.3"
```
Comment (padmkris123) [inline swc-uxp-starter/package.json:35]: Please add the explanation for this in the README

Comment (bhatiaPankaj) [reply swc-uxp-starter/package.json:35]: The components added (shared, styles, base) are not UI components but are required to be in the resolution block just as the `theme` for version consistency. As we have not mentioned the theme inclusion in the resolution block in the README, I have updated the screenshot with these umbrella components added.

Comment (padmkris123) [reply swc-uxp-starter/package.json:35]: And these are going to be common to all components, right?

Comment (bhatiaPankaj) [reply swc-uxp-starter/package.json:35]: Yes, they are common for all the components

**Pull request closed without merging**